CLA_SIGNERS.yaml

```diff
@@ -289,6 +289,13 @@ companies:
         email: nidmgg@gmail.com
         github: nidmgh
 
+  - name: Informatica
```

Comment:
Not that important, but shouldn't _Informatica_ be added after _IBM_?

Apart from that, I'm not quite sure what to review in cases like these. I didn't see the CLA mail, so I can't really verify the information here.

Reply:
> Not that important, but shouldn't _Informatica_ be added after _IBM_?

You're right; fixed!

> Apart from that, I'm not quite sure what to review in cases like these. I didn't see the CLA mail, so I can't really verify the information here.

You're right again; point taken. I'll wait for @pluradj to review.

Thanks again!